lib/filerotator.js

```diff
@@ -46,7 +46,11 @@ function FileRotator(logpath, totalFiles, totalSize, gzip) {
         .pipe(zlib.createGzip())
         .pipe(fs.createWriteStream(zippedPath))
         .on('finish', function () {
```

Comment:
To make sure the underlying operating system has processed all the data
```suggestion
        .on('close', function () {
```